extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.json

```diff
@@ -0,0 +1,92 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFields",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Step Event Action Execution Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEvent",
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "This Mixin will have Journey Action Metadata Fields",
+  "definitions": {
+    "journeyStepEventActionExecutionFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Orchestration for creating user journey.",
+          "type": "object",
+          "properties": {
+            "journey:actionId":{
+              "title":"actionId",
+              "description":"Id of the action that is being executed",
+              "type":"string"
+            },
+            "journey:actionName":{
+              "title":"actionName",
+              "description":"Name of the action If no name has been set, the stepName will be taken",
+              "type":"string"
+            },
+            "journey:actionType":{
+              "title":"actionType",
+              "description":"Type of the action",
```

Comment:
Is there a fixed list of these or open ended?

Reply:
for now, these are the fields we are interested in to pass to experience platform from Journey Orchestration. but in future, if new fields need to be passed will update this